core/harambe_core/parser/currencies.py

```diff
@@ -0,0 +1,113 @@
+all_currencies = sorted(
```

Comment:
Why is this a list and not a dictionary?

Also instead of sorting it everytime, we should just hardcode in sorted order.

Reply:
good call, was to avoid a headeache each time i will add a new currency, will make it dict here for more speed @awtkns